samples/cpp/resize_pil.cpp

```diff
@@ -0,0 +1,32 @@
+#include <opencv2/opencv.hpp>
+//Include file for every supported OpenCV function
+
+using namespace cv;
+
+int main(int argc, char** argv)
+{
+  Mat image;
+  image = imread("d:\\mask.bmp", IMREAD_GRAYSCALE);  // Read the file
+
+  if (!image.data)  // Check for invalid input
+  {
+    std::cout << "Could not open or find the image" << std::endl;
+    return -1;
+  }
+  imshow("Source image", image);
+
+  Mat target_image;
+
+  resize(image, target_image, Size(16, 16), 0, 0, INTER_NEAREST_PIL);
+  for (int row = 0; row < target_image.rows; ++row) {
+      for (int col = 0; col < target_image.cols; ++col) {
+        std::cout << abs((int) target_image.at<char>(row, col)) << ' ';
+      }
+      std::cout << "\n";
+  }
+  resize(target_image, image, Size(256, 256), 0, 0, INTER_NEAREST_PIL);
+
+  imshow("Target image", image);
```

Comment:
Perhaps it makes sense to show results of both `NEAREST_PIL` and `NEAREST` to show difference, otherwise the sample does not have any value. Or, even better, all modes and flags should be visualized with `lena.jpg` and/or some synthetic images - that would be a great example.

Reply:
Maybe it's even better to have Python sample instead of C++ which can run both  PIL and OpenCV and compare the difference between, INTER_NEAREST, potential INTER_NEAREST_PIL and PIL?

Reply:
Yes, sure. I've created resize_pil.py in which I've demonstrated:
1. Difference between cv.INTER_NEAREST and PIL NEAREST
2. Equivalence betwen cv.INTER_NEAREST_PIL and PIL NEAREST 
I've used lena.jpg